Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build stage
+ARG PYTHON_VERSION=3.8-slim
+FROM python:${PYTHON_VERSION} AS build
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY . /app
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Run stage
+FROM build AS runtime
+
+EXPOSE 8080
+CMD ["python", "manage.py", "migrate"]
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
There are two CMD instructions in this Dockerfile. Only the last CMD instruction will be executed, which means the migration command will not run. Consider combining these commands into a single CMD instruction or using a script to handle both operations.